zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -472,7 +472,14 @@ public void run(String paragraphId) {
         logger.debug("New paragraph: {}", pText);
         p.setEffectiveText(pText);
       } else {
-        throw new InterpreterException("Interpreter " + requiredReplName + " not found");
+        InterpreterException intpException;
+        intpException = new InterpreterException(
```

Comment:
Probably a nitpick, but declaration and definition of this variable can be done on a single line.